dracut/10usr-generator/module-setup.sh

```diff
@@ -12,4 +12,18 @@ install() {
         "$systemdutildir/system-generators/usr-generator"
     inst_simple "$moddir/remount-sysroot.service" \
         "$systemdutildir/system/remount-sysroot.service"
+
+    # Overwrite systemd-fsck-usr.service because it wants After=local-fs-pre.target
+    # which prevents (the systemd-generated) sysusr-usr.mount from starting before Ignition
+    # but we need it because we want to start Ignition from /sysusr/usr/ already.
+   cat > "$initdir/etc/systemd/system/systemd-fsck-usr.service" <<EOF
+# Set up by 10usr-generator/module-setup.sh
+[Unit]
+DefaultDependencies=no
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStart=true
```

Comment:
I'm wondering if this should be kept more-or-less in sync with systemd - upstream seems to have more stuff in the unit.

Reply:
Or maybe we should carry a patch to systemd, so no overwriting needs to be done here?

Reply:
This is a minimal unit, to have it start instead of masking it. What we would need upstream is a way of resetting the After= dependencies through a drop-in…